src/pyhf/pdf.py

```diff
@@ -567,38 +597,94 @@ def _create_nominal_and_modifiers(self, config, spec):
         return mega_mods, _nominal_rates
 
     def expected_auxdata(self, pars):
+        """
+        Compute the expected value of the auxiliary measurements.
+
+        Args:
+            pars (`tensor`): The parameter values
+
+        Returns:
+            Tensor: The expected auxiliary data
+
+        """
         return self.make_pdf(pars).pdfobjs[1].expected_data()
 
     def _modifications(self, pars):
         return self.main_model._modifications(pars)
 
     @property
     def nominal_rates(self):
+        """Nominal value of bin rates of the main measurment."""
         return self.main_model.nominal_rates
 
     def expected_actualdata(self, pars):
+        """
+        Compute the expected value of the main measurements.
+
+        Args:
+            pars (`tensor`): The parameter values
+
+        Returns:
+            Tensor: The expected main data
+
+        """
         return self.make_pdf(pars).pdfobjs[0].expected_data()
 
     def expected_data(self, pars, include_auxdata=True):
+        """
+        Compute the expected value of the main measurements.
+
+        Args:
+            pars (`tensor`): The parameter values
+
+        Returns:
+            Tensor: The expected main data
```

Comment:
How is this different from `expected_actualdata` since they both "return" the same thing?